site/ko/tutorials/distribute/multi_worker_with_estimator.ipynb

```diff
@@ -0,0 +1,395 @@
+    "##### Copyright 2019 The TensorFlow Authors.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "tuOe1ymfHZPu"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "# https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "MfBg1C5NB3X0"
+   },
+   "source": [
+    "# 추정기(Estimator)를 사용한 다중 워커(Multi-worker) 훈련\n",
+    "\n",
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/distribute/multi_worker_with_estimator\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />TensorFlow.org에서 보기</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/ko/tutorials/distribute/multi_worker_with_estimator.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />구글 코랩(Colab)에서 실행하기</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/ko/tutorials/distribute/multi_worker_with_estimator.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />깃허브(GitHub) 소스 보기</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 [공식 영문 문서](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/distribute/multi_worker_with_estimator.ipynb)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 [tensorflow/docs](https://github.com/tensorflow/docs) 깃허브 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 참여하려면 [docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ko)로 메일을 보내주시기 바랍니다.\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "xHxb-dlhMIzW"
+   },
+   "source": [
+    "## 개요\n",
+    "\n",
+    "Note: `tf.distribute` API와 함께 추정기를 사용할 수는 있지만, `tf.distribute`와 함께 케라스(Keras)를 사용하는 것을 추천합니다. [케라스를 사용한 다중 워커(Multi-worker) 훈련](../../guide/multi_worker_with_keras.ipynb)을 봐주세요. `tf.distribute.Strategy`를 추정기와 사용하는 것은 부분적으로만 지원하고 있습니다.\n",
```

Comment:
I think '다중 워커(Multi-worker)' makes reader confused(korean + english).
So, I suggest '멀티 워커' or '다중 작업자' rather than '다중 워커'.
How about you?

Reply:
There were some suggestions regarding these translation word selection in previous work ( https://github.com/tensorflow/docs/pull/808 ).
I've decided `워커` because there are several places we should enter "worker", and it will be better not to translate this word. On the other hand, 'multi' is used in other places, for example 'multi-CPU', 'multi-GPU', and I felt that `다중` is a *little* bit easier than `멀티` for Korean readers. So the outcome of these choices is `다중 워커`. I agree with you that 'Korean+English' combination is somewhat awkward, but I think that keeping consistency with other terms would be better here.